src/runtime_src/core/pcie/windows/core_system.cpp

```diff
@@ -17,7 +17,9 @@
 
 #include "common/core_system.h"
 #include "gen/version.h"
-#include <atlstr.h>
+#include <windows.h>
+
+#define BUFFER 128
```

Comment:
Thank you for "defining" the magic variables.  
With you next submission, could you move it closer to where it is used (e.g., inside of the get_os_info() function).  

Something to think about
You might wish to define this value as a "static const unsigned int" inside of the method.   Doing so will allow anyone debugging this code to easily  determine its value.